### ISSUE TITLE
Advanced search logic is not working

### ISSUE THREAD

**Issue by JustACuteGirlLookingForAManToFixHerBugs** [talengu/Wox.Plugin.BookmarkSearcher#8]
Hey! I'm not sure how the search is handled for plugins, but I've noticed that other Bookmarks Plugin is utilizing the more advanced search, while this doesn't. Is it something I've missed?
![image](https://user-images.githubusercontent.com/9111191/77256864-72239a80-6c79-11ea-887f-de6f172f19eb.png)
![image](https://user-images.githubusercontent.com/9111191/77256866-73ed5e00-6c79-11ea-9b15-4e583e82e13d.png)
![image](https://user-images.githubusercontent.com/9111191/77256870-75b72180-6c79-11ea-9272-1fb36a893e7c.png)

**Comment by talengu** [comment]
Yes, this plugin's search method does not support this situation.
Nice, I will add some fuzzy search functions  when I was free.
